src/nameserver/system_table.h

```diff
@@ -33,22 +33,27 @@ constexpr const char* INTERNAL_DB = "__INTERNAL_DB";
 constexpr const char* JOB_INFO_NAME = "JOB_INFO";
 constexpr const char* PRE_AGG_META_NAME = "PRE_AGG_META_INFO";
 
+
+constexpr const char* INFORMATION_SCHEMA_DB = "INFORMATION_SCHEMA";
```

Comment:
why add new database

Reply:
> why add new database

GLOBAL_VARIABLES in mysql is stored in db INFORMATION_SCHEMA

Reply:
we're creating table `information_schema.global_variables` the same as mysql